build/plugins/accordion.html

```diff
@@ -0,0 +1,208 @@
+<!-- accordion plugin -->
+
+<script>
+    ///////////////////////////
+    // DESCRIPTION
+    ///////////////////////////
+
+    // This Manubot plugin allows sections of content under <h2> headings to be collapsible.
+
+    (function() {
+        ///////////////////////////
+        // ASSETS
+        ///////////////////////////
+
+        // html svg of arrow icon
+        // modified from: https://fontawesome.com/icons/angle-down
+        let arrowIcon =
+            "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 448 512'><path fill='currentColor' d='M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z'></path></svg>";
```

Comment:
I'm not sure how to elegantly do this, because most of the length comes from the long svg path `d=` parameter which defines the point of the path. I suppose I can split it up arbitrarily in the spaces of the path description, or if you know any automatic formatter tools to do this.

Reply:
This is just a long string in JS. Please feel free to cut it anywhere you feel appropriate. I don't have any preferences once each line is shorted than 80 characters.

Reply:
> I don't have any preferences once each line is shorted than 80 characters.

Hmm. I'm not sure I'm in favor of such short line lengths. The 80 character max is really an outdated concept.

For these SVGs that we're tracking with git, I think it makes sense to have the entire points of the path on a single line. This seems like the most semantic way to break the line. Similar to how we do one-sentence-per-line in our markdown prose. I think we should avoid lines over 120 characters when possible, unless that requires breaking something line this `d=` list or a URL, etcetera.

Reply:
80 is the standard for javascript these days. However it generally doesn't apply to long continuous strings or comments, just logic. What we have here is a long string. Even though it contains structured content, I don't think it looks good to split up with `+`'s.

What about moving the assets outside of the script tag so they can just be plain html, like this:

```html

<script>
// DESCRIPTION
// this manubot plugin blah blah

// OPTIONS

...plugin code...
</script>

// modified from: https://fontawesome.com/icons/
<svg id="left_arrow_icon" viewBox="0 0 512 512">
    <path
        fill="blah"
        d="1 09287129 8712 172e91872e 97812 987"
    >
    </path>
</svg>
```